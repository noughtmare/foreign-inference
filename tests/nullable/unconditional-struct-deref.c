typedef struct {
  int *x;
} point;

int f(point *p, int b) {
  if (b) {
    return *p->x;
  }
  return 0;
}
