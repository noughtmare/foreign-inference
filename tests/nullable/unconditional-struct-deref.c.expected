fromList [("%struct.point", fromList ["x"])]
